taca/utils/bioinfo_tab.py

```diff
@@ -60,12 +60,7 @@ def update_statusdb(run_dir):
     view = db.view('latest_data/sample_id')
     #Construction and sending of individual records, if samplesheet is incorrectly formatted the loop is skipped
     for flowcell in project_info:
-        if flowcell == 'UNKNOWN':
-            obj={'run_id':run_id, 'project_id':'ERROR_Samplesheet'}
-            logger.info("INVALID SAMPLESHEET, CHECK {} FORMED AT {}".format(run_id, valueskey))
-            error_emailer('no_samplesheet', run_id)
-            db.save(obj)
-        else:
+        if not flowcell == 'UNKNOWN':
```

Comment:
This reg ex at line 33 `  rundir_re = re.compile("\d{6}_[ST-]*\w+\d+_\d+_[AB]?[A-Z0-9\-]+")` is not bullet proof... there is no reason why we cannot have an instrument with a name like `M12K12`, in this case the part `[ST-]*\w+\d+` will fail as it is expecting a list of letters followed by a list of numebrs